src/main/java/org/sagebionetworks/web/client/widget/table/modal/fileview/SqlDefinedTableEditor.java

```diff
@@ -44,25 +51,28 @@ public SqlDefinedTableEditor(
     this.synAlert = synAlert;
     this.view.setPresenter(this);
     view.setSynAlert(synAlert);
-    view.setHelp(
-      MATERIALIZED_VIEW_HELP_MARKDOWN,
-      CreateTableViewWizard.VIEW_URL
-    );
   }
 
   public SqlDefinedTableEditor configure(
     String parentEntityId,
     EntityType entityType
   ) {
-    if (
-      !EntityType.materializedview.equals(entityType) &&
-      !EntityType.virtualtable.equals(entityType)
-    ) {
+    if (EntityType.materializedview.equals(entityType)) {
+      view.setHelp(
+        MATERIALIZED_VIEW_HELP_MARKDOWN,
+        CreateTableViewWizard.VIEW_URL
```

Comment:
I spent some time trying to improve this in https://sagebionetworks.jira.com/browse/DOCS-117 but these new docs must be reviewed and published before we should reference them.